frameworks/elastic/universe-kibana/config.json

```diff
@@ -14,6 +14,12 @@
           "description": "The user that runs the Kibana services and owns the Mesos sandbox.",
           "type": "string",
           "default": "nobody"
+        },
```

Comment:
Does this have any effect for Kibana as it's just a Marathon app and does not use the SDK?